os-autoinst-obs-auto-submit

```diff
@@ -6,8 +6,8 @@ export LC_ALL=C
 src_project=${src_project:-devel:openQA}
 dst_project=${dst_project:-${src_project}:tested}
 staging_project=${staging_project:-${src_project}:testing}
-submit_target="${submit_target:-"openSUSE:Factory"}"
-submit_target_escaped=$(echo -n "$submit_target" | sed -e 's|\:|_|g') # avoid, e.g. "repoid 'openSUSE:Factory' is illegal"
+factory_target="${factory_target:-"openSUSE:Factory"}"
+factory_target_escaped=$(echo -n "$factory_target" | sed -e 's|\:|_|g') # avoid, e.g. "repoid 'openSUSE:Factory' is illegal"
```

Comment:
Please keep the original names intact. And then what you add is "secondary" or "additional" targets

Reply:
I will but it might be confusing. it was for me at least to follow the logic.